### PR TITLE
Add Theme switcher

### PR DESCRIPTION
**Pull request by JaneEllison**
Add theme switcher component: add styles, theme constants, saving theme in local storage, backgrounds of cityForecast layout, witch depends on current theme. 
No Redux at this time.

### CODE REVIEW

File: src/layouts/CityForecast/CityForecast.jsx [hunk 1]
```diff
@@ -1,20 +1,27 @@
...
 import TemperatureUniteToggle from './TemperatureUniteToggle/TemperatureUniteToggle';
 import VerticalLine from '../../components/VerticalLine/VerticalLine';
 
-function CityForecast({ cityForecast, cityInfo }) {
+function CityForecast({ cityForecast, cityInfo, theme }) {
   const weather = cityForecast.current;
   const symbolPhrase = weather.symbolPhrase[0].toUpperCase() + weather.symbolPhrase.slice(1);
+  const themeBg = theme === THEMES.light ? lightBg : darkBg;
+
+  console.log(theme);
```
Comment (anton-vlasik) [inline src/layouts/CityForecast/CityForecast.jsx]: remove console.logs

File: src/layouts/Header/Header.jsx [hunk 2]
```diff
@@ -5,13 +5,13 @@ import NavBar from './NavBar/Navbar';
...
 
-function Header() {
+function Header({ theme, onSwitchTheme }) {
   return (
     <div className={styles.header}>
       <NavBar />
       <div className={styles.wrapper}>
         <SearchInput />
-        <ThemeSwitcher />
+        <ThemeSwitcher theme={theme} onSwitchTheme={onSwitchTheme} />
```
Comment (anton-vlasik) [inline src/layouts/Header/Header.jsx]: Optional: onSwitch would probably be enough here

File: src/layouts/CityForecast/CityForecast.jsx (hunk 1 above)
Comment (JaneEllison) [reply src/layouts/CityForecast/CityForecast.jsx]: done

File: src/layouts/Header/Header.jsx (hunk 2 above)
Comment (JaneEllison) [reply src/layouts/Header/Header.jsx]: I agree with you, sounds better

**Review by anton-vlasik — APPROVED** [review]

**Review by ximet — APPROVED** [review]

**Pull request merged**